HealthcareSystemFrontends/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -209,6 +218,52 @@ public MainWindow(string currentUserRole)
 
             RestrictUsersAccessBasedOnRole();
 
+            SetDataToUIControls();
+        }
+
+        public void SetDataToUIControls()
+        {
+            SetValuesInDoctorSpecialtyComboBoxForAppointmentSearch();
+            SetValuesInPatientComboBoxForAppointmentSearch();
+            SetSelectableEquipmentForAppointmentSearch();
+        }
+
+
+        public void SetValuesInDoctorSpecialtyComboBoxForAppointmentSearch()
+        {
+            DoctorService doctorService = new DoctorService();
+            List<SpecialtyDTO> allDoctorSpecialties = doctorService.GetAllSpecialties();
+
+            AppointmentDoctorSpecializationComboBox.ItemsSource = allDoctorSpecialties;
+        }
+
+        public void SetValuesInPatientComboBoxForAppointmentSearch()
+        {
+            PatientService patientService = new PatientService();
+            List<PatientBasicDTO> allPatients = patientService.GetAllPatients();
+
+            AppointmentPatientComboBox.ItemsSource = allPatients;
+        }
+
+        public void SetSelectableEquipmentForAppointmentSearch()
+        {
+            AllEquipmentTypes = new ObservableCollection<EquipmentTypeForViewDTO>();
+
+            EquipmentTypeService equipmentTypeService = new EquipmentTypeService();
+            List<EquipmentTypeDTO> equipmentTypes = equipmentTypeService.GetEquipmentTypes();
+
+            foreach(EquipmentTypeDTO equipmentType in equipmentTypes)
+            {
+                AllEquipmentTypes.Add(new EquipmentTypeForViewDTO(equipmentType, false));
+            }
+        }
+
+        private void CheckBox_Checked(object sender, RoutedEventArgs e)
+        {
+            foreach (EquipmentTypeForViewDTO eas in AllEquipmentTypes)
+            {
+                Console.WriteLine(eas.IsSelected);
+            }
         }
```

Comment:
I guess this is for debugging purposes, do you still need it or did you forget to delete?

Reply:
I will delete it in the next PR after we connect the frontend and backend for appointments.